pkgs/misc/emulators/ryujinx/updater.sh

```diff
@@ -1,10 +1,32 @@
-#!/usr/bin/env nix-shell
-#!nix-shell -i bash -p curl jq common-updater-scripts dotnet-sdk_5
+#! /usr/bin/env nix-shell
+#! nix-shell -i bash -p coreutils gnused curl common-updater-scripts nix-prefetch-git jq dotnet-sdk_5
 set -eo pipefail
 cd "$(dirname "${BASH_SOURCE[0]}")"
 
 deps_file="$(realpath "./deps.nix")"
 
+nix-prefetch-git https://github.com/ryujinx/ryujinx --quiet > repo_info
+new_hash="$(cat repo_info | jq -r ".sha256")"
+new_rev="$(cat repo_info | jq -r ".rev")"
```

Comment:
This is normally saved into a variable.

Reply:
Do you mean the `nix-prefetch-git` command? If I do that, we need to run it several times which takes quite a while, as it refetches the entire repository. This is why i saved the output to a file, as we can simply re-use it since it outputs both the upstream sha256sum and revision.

Reply:
I meant a bash variable but does not matter that much.